mxcube3/ui/components/SampleGrid/SampleGrid.css

```diff
@@ -8,7 +8,9 @@
   display: inline;
 }
 
-.samples-grid{ 
+.samples-grid{
```

Comment:
Not so certain about this, I think we prefer when the page scrolls at least until now. Why did you limit the height ?

Reply:
I did this way so the top bar and filter row are always visible. If not, it is quite annoying when many samples are loaded.

Reply:
Top bar and filter should always be visible, with the floating bar feature

Reply:
then it stopped working, at least for me. Could you confirm that it works for you?

Reply:
Yes, it seems like that feature is broken we have to take a look at it. Its maybe some CSS that changed